geode-core/src/main/java/org/apache/geode/distributed/internal/OperationExecutors.java

```diff
@@ -26,7 +26,7 @@
  */
 public interface OperationExecutors {
   int MAX_THREADS =
-      Integer.getInteger("DistributionManager.MAX_THREADS", 100);
+      Integer.getInteger("DistributionManager.MAX_THREADS", 300);
```

Comment:
There is another ticket to address this issue: GEODE-8357
This change has an impact on overall system requirement and performance; it will be nice to send a proposal to developer community. This helps to get feedback from engineers who had encountered issue with this setting and suggested the recommendation to the users (support teams); there was conversation about changing this to big number like 1000. 
It will be good to come up with a right number based on previous experience in setting this number.

We also need to think about setting the system properties automatically; some kind of auto-tuning service in the product that will adopt/changes the value based on the resource availability and the system load.

Reply:
@agingade we created this PR after receiving feedback from customers and support. This is one of the requested features from the field and these are the numbers we have received from support.
We are also aligning these numbers with MAX_FE_THREADS, which was changed to cpu*32 to tune the internal tests.

This can be a temporary band-aid till we start on GEODE-8357, which has more extensive changes and we can involve the community feedback.

Reply:
Having talked to some other people with more hands-on experience with hitting the limit currently set by `MAX_THREADS`, a value of 1000 was decided to be more appropriate. The PR has been updated to reflect this.